### ISSUE TITLE
[Quest] Shutting Down Manaforge Coruu - Coruu Control Cnsole swings at player

### ISSUE THREAD

**Issue by dieing4me** [chromiecraft/chromiecraft#4948]
### What client do you play on?

enUS

### Faction

Alliance

### Content Phase:

65-69

### Current Behaviour

After initiating the shutdown during the quest, the Console becomes hostile but untargetable and will swing and damage the player, making the quest more difficult to complete than intended.

![Wow_feDb1NqZAx](https://user-images.githubusercontent.com/29512498/217989984-ff482f5a-5c8b-4631-aa0c-34a1b779909f.png)

### Expected Blizzlike Behaviour

The player defends the console for 2 minutes while be assaulted by technicians that spawn every so often, without being assaulted by the console itself

### Source

Video of the quest:
https://youtu.be/-SeHU1Foyz8

### Steps to reproduce the problem

Get the quest Shutting Down the Manaforge
Go to 48.95 , 81.46 and kill Overseer Seylanna
Pick up the Coruu Access Crystal
Right click and interact with the Coruu Control Console and left click "Begin emergency shutdown."
Defend the console while examining the combat logs and seeing the Coruu Control Console hit the player for physical damage

### Extra Notes

Quest: https://wowgaming.altervista.org/aowow/?quest=10321
NPC Kill Target (Overseer Seylanna): https://wowgaming.altervista.org/aowow/?npc=20397
Item from NPC (Coruu Access Crystal): https://wowgaming.altervista.org/aowow/?item=29396
Control Console: https://wowgaming.altervista.org/aowow/?npc=20417

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/f3f2364befeda2fe09fb495e2cb81f2974e6f59e

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by dieing4me** [comment]
The quest is still completable if one successfully defends it from the spawning mobs however even afterwards the console was still swinging at me afterwards and further still after Shadowmelding. I had to run out of the interior of the manaforge and disengage via distancing. I ran back inside to see if it was still there and it seems so because it started attacking me and I had to disengage via distancing again. The Control Console aggroing and hitting me didn't trigger combat but it would still prevent me from leaving combat if engaged with other mobs while it was actively attacking.

In short; the console stays active and is hostile to the player and will attack during and after the quest.

**Comment by balleny** [comment]
this issue applies to all mana forges. it becomes even more apparent, if you have a pet out, as those are typically attacked first by newly spawned npcs / triggers

**Comment by foo-17** [comment]
tested with AzerothCore rev. 68263b773daa 2023-11-07 (master branch).

tested Aldor and Scryer version of this quest (quests with ids 10321, 10330)
![image](https://github.com/chromiecraft/chromiecraft/assets/150605164/3e1ff715-eed6-4b3a-b137-010488944e03)

tested as Alliance, level 255 and level 69. could not replicate this issue. Once quest was started, technicians appeared and attacked, but console did not attack me. In combat log, tab "what happened to me", I am not attacked by console throughout the event (screenshot from trial as Ally, level 69, scryer version of the quest (id: 10330)).
![image](https://github.com/chromiecraft/chromiecraft/assets/150605164/ae28efb0-0762-4232-872a-8a8d612a471d)

**Comment by The-GhostRider** [comment]
[![Anteprima](https://user-images.githubusercontent.com/1884642/287841708-03f0fb15-7c82-4f9d-a641-c25142c813b9.png)](https://www.youtube.com/embed/YfKansM4gb0)
So..
I was unable to reproduce the problem (as show my video up)..
In fact I went to check the NPC trigger flags: [Coruu Control Console](https://wowgaming.altervista.org/aowow/?npc=20417)
And I must say that he has all the correct flags and should not attack Players..
But there is the remote possibility that due to strange dynamics that escape me now, these flags are sometimes ineffective, so I inserted an additional flag (for safety) making the NPC peaceful.

![Screenshot_2](https://github.com/chromiecraft/chromiecraft/assets/1884642/2dcd3126-1732-4b6a-bc3c-b3eb4816918b)


This way it is 100% sure that the NPC does not attack any player.
